common/lib/xmodule/xmodule/library_content_module.py

```diff
@@ -191,13 +191,15 @@ def _publish_event(self, event_name, result, **kwargs):
             "result": result,
             "previous_count": getattr(self, "_last_event_result_count", len(self.selected)),
             "max_count": self.max_count,
+            "user_id": user_id,
+            "course_id": unicode(self.course_id),
```

Comment:
I may be incorrect, but I thought this data needed to be in the "context" part of the event, and not in the "payload".  Didn't the fix for the grading events require using a tracking context-manager so the missing info went into the event's "context"?